packages/flutter/test/rendering/mock_canvas.dart

```diff
@@ -527,14 +527,27 @@ class _TestRecordingCanvasPaintsNothingMatcher extends _TestRecordingCanvasMatch
 
   @override
   bool _evaluatePredicates(Iterable<RecordedInvocation> calls, StringBuffer description) {
-    if (calls.isEmpty)
+    final Iterable<RecordedInvocation> paintingCalls = _filterCanvasCalls(calls);
+    if (paintingCalls.isEmpty)
       return true;
     description.write(
       'painted something, the first call having the following stack:\n'
-      '${calls.first.stackToString(indent: "  ")}\n'
+      '${paintingCalls.first.stackToString(indent: "  ")}\n'
     );
     return false;
   }
+
+  static const List<Symbol> _nonPaintingOperations = const <Symbol> [
+    const Symbol('save'),
```

Comment:
rather than calling the Symbol constructor, use literals. `#save` and `#restore`.

Reply:
done